index.html

```diff
@@ -1487,745 +1571,61 @@ <h3>DID Controller</h3>
-href="#security-considerations"></a> for information related to threat models
-and attack vectors.
-        </p>
-      </section>
-
-      <section>
-        <h3>Also Known As</h3>
-
-        <p>
-A <a>DID subject</a> can have multiple identifiers for different purposes, or
-at different times. The assertion that two or more <a>DIDs</a> (or other types
-of <a>URI</a>) refer to the same <a>DID subject</a> can be made using the
-<code><a>alsoKnownAs</a></code> property.
-        </p>
-
-        <dl>
-          <dt><dfn>alsoKnownAs</dfn></dt>
-          <dd>
-The <code>alsoKnownAs</code> property is OPTIONAL. If present, the value MUST
-be a <a data-cite="INFRA#ordered-set">set</a> where each item in the
-set is a <a>URI</a> conforming to [[RFC3986]].
-          </dd>
-          <dd>
-This relationship is a statement that the subject of this identifier is
-also identified by one or more other identifiers.
-          </dd>
-        </dl>
-
-        <div class="note" title="Equivalence and alsoKnownAs">
-          <p>
-Applications might choose to consider two identifiers related by
-<code><a>alsoKnownAs</a></code> to be equivalent <em>if</em> the
-<code><a>alsoKnownAs</a></code> relationship is reciprocated in the reverse
-direction. It is best practice <em>not</em> to consider them equivalent in the
-absence of this inverse relationship. In other words, the presence of an
-<code><a>alsoKnownAs</a></code> assertion does not prove that this assertion
-is true. Therefore, it is strongly advised that a requesting party obtain
-independent verification of an <code>alsoKnownAs</code> assertion.
-          </p>
-          <p>
-Given that the <a>DID subject</a> might use different identifiers for different
-purposes, an expectation of strong equivalence between the two identifiers, or
-merging the information of the two corresponding <a>DID documents</a>, is not
-necessarily appropriate, <em>even with</em> a reciprocal relationship.
-          </p>
-        </div>
-
       </section>
 
     </section>
 
     <section>
       <h2>Verification Methods</h2>
       <p>
-A <a>DID document</a> can express <a>verification methods</a>, such as
-cryptographic public keys, which can be used to <a>authenticate</a> or authorize
-interactions with the <a>DID subject</a> or associated parties. For example, a
-cryptographic public key can be used as a <a>verification method</a> with
-respect to a digital signature; in such usage, it verifies that the signer
-could use the associated cryptographic private key. <a>Verification methods</a>
-might take many parameters. An example of this is a set of five cryptographic
-keys from which any three are required to contribute to a cryptographic
-threshold signature.
+A <a>DID document</a> can express <a>verification methods</a>, as defined
+<a data-cite="CID#verification-methods">Section 2.2: Verification Methods</a> of
+[[[CID]]]. Identifiers used in <a>verification methods</a> can be expressed
```

Comment:
```suggestion
[[[CID]]]. Identifiers used in <a>verification methods</a> MUST be expressed
```

Unless I am misunderstanding this alignment with the CID spec, I believe this is a normative requirement of the DID spec